tests/smoketest/minimal-collectd.conf.template

```diff
@@ -12,7 +12,7 @@ LoadPlugin amqp1
 <Plugin "amqp1">
   <Transport "name">
     Host "default-interconnect"
-    Port "5672"
+    Port "5671"
```

Comment:
I thought 5671 was for the internal QDR connections?

Reply:
Yea I didn't look through this and it was probably just late when I looked at this. I believe you :D

Reply:
> I thought 5671 was for the internal QDR connections?

According to our wiki page: 

```
5671 | edge, Anonymous SASL, SSL (profile: openstack) - Used for connections from OSP
5672 | edge, no auth, no ssl - Used by legacy SG
```

However AFAICT this port is being used without SSL in these tests. Is our wiki wrong? I don't have a live system in front of me to check.